airgun/widgets.py

```diff
@@ -225,6 +225,75 @@ def read(self):
         return values
 
 
+class DatePicker(TextInput):
```

Comment:
yeah, it seems completely different to patternfly one. Anyway, did you check that widgetastic datepicker is not usable for us?

Reply:
inherited from `TextInput`? Okay then rename class to `DatePickerInput` as it is not actually datepicker

Reply:
widgetastic's one has visually the same input, but totally different calendar. Yep, inherited from TextInput as its skeleton works better for me than regular `Widget`, less things to redefine.